skimage/restoration/_nl_means_denoising.pyx

```diff
@@ -394,12 +388,12 @@ cdef inline double _integral_to_distance_2d(IMGDTYPE [:, ::] integral,
     distance = max(distance, 0.0) / h2s2
     return distance
 
-
+# @cython.wraparound(False)
 @cython.cdivision(True)
 @cython.boundscheck(False)
-cdef inline double _integral_to_distance_3d(IMGDTYPE [:, :, ::] integral,
```

Comment:
Is there a reason why the decorators were left here, and the commented one added?

Reply:
good catch.  that was unintentional.